README.md

```diff
@@ -1,15 +1,19 @@
-# Arbitrum Nitro Contracts
+# Arbitrum Nitro Rollup Contracts
 
 This is the package with the smart contract code that powers Arbitrum Nitro.
 It includes the rollup and fraud proof smart contracts, as well as interfaces for interacting with precompiles.
 
-For more information see https://developer.arbitrum.io/docs/public_nitro_devnet
+For more information see https://developer.arbitrum.io/intro
```

Comment:
Is there a better link to use here? I don't think the old one makes sense now that nitro is live.